tests/foreman/api/test_environment.py

```diff
@@ -8,8 +8,9 @@
 from ddt import ddt
 from fauxfactory import gen_string
 from nailgun import entities
+from nailgun.entity_mixins import _get_entity_ids
```

Comment:
I am not comfortable calling a function prefixed with _ If this function is to be usable by other modules, please make it public. Prefixed with _ has a reason and I dont want us to change it. Non-blocker for this PR though